plaso/storage/sqlite/sqlite_file.py

```diff
@@ -237,7 +239,7 @@ def _CountStoredAttributeContainers(self, container_type):
       raise ValueError('Attribute container type {0:s} is not supported'.format(
           container_type))
 
-    if not self._HasTable(self._CONTAINER_TYPE_ANALYSIS_REPORT):
```

Comment:
Maybe fix this in a separate PR and add test coverage.

Reply:
Done - #2402